docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -466,6 +466,35 @@
         like "python" or "java" - that specifically define memory uses for
         those targets.</para>
       </sect3>
+
+      <sect3 id="CNT_ESPandECLWatch">
+        <title>ESP and ECLWatch </title>
+
+        <para>ECL Watch works much the same way deployed via bare metal or a
+        container. There are a few exceptions. </para>
+
+        <sect4 id="CNTR_ZAP">
```

Comment:
The content is good but it feels out of place in the context of the chapter since it doesn't cover configuration settings. . Perhaps it should go in the Logging chapter.